components/CopyNotifier.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const CatNotifer: React.FC = () => {
+  return <div className="copy-notifer">Copied Cat Id</div>;
+};
+
+export default React.memo(CatNotifer);
```

Comment:
unless this is my lack of next.js experience speaks me, we don't need to wrap this in memo.